changemakers/frappe_changemakers/doctype/payment_type/payment_type.json

```diff
@@ -0,0 +1,55 @@
+{
+ "actions": [],
+ "allow_rename": 1,
+ "creation": "2023-10-14 12:52:50.195141",
+ "default_view": "List",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "consultation",
+  "testing",
+  "medicine"
+ ],
+ "fields": [
+  {
+   "fieldname": "consultation",
+   "fieldtype": "Data",
+   "label": "Consultation"
+  },
+  {
+   "fieldname": "testing",
+   "fieldtype": "Data",
+   "label": "Testing"
+  },
+  {
+   "fieldname": "medicine",
+   "fieldtype": "Data",
+   "label": "Medicine"
+  }
+ ],
```

Comment:
Just a `name` field should be enough. It should be `data` type field. 

What the [issue](https://github.com/frappe/changemakers/issues/130#issuecomment-1740692205) meant was, the doctype entries should be available as a drop down.

For reference, check out the implementation of Gender field in `Beneficiary` doctype. Options for Payment Type can be created as entries of `Payment Type` doctype and then they can be exported using [fixtures](https://www.youtube.com/watch?v=fFEpiZbNjB0).